fn_ioc_parser/fn_ioc_parser/util/ioc_parser_helper.py

```diff
@@ -0,0 +1,152 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+import os
+import tempfile
+from docx import Document
+import io
+from pdfminer.converter import TextConverter
+from pdfminer.pdfinterp import PDFPageInterpreter
+from pdfminer.pdfinterp import PDFResourceManager
+from pdfminer.pdfpage import PDFPage
+import xlrd
+
+
+class IOCParserHelper(object):
+    def __init__(self):
+        # Resilient Rest Api calls url's
+        self.ARTIFACT_META_DATA_URL = "/incidents/{0}/artifacts/{1}"
+        self.ARTIFACT_DATA_URL = "/incidents/{0}/artifacts/{1}/contents"
+        self.ATTACHMENT_META_DATA_URL = "/incidents/{0}/attachments/{1}"
+        self.ATTACHMENT_DATA_URL = "/incidents/{0}/attachments/{1}/contents"
+        self.TASK_META_DATA_URL = "/tasks/{0}/attachments/{1}"
+        self.TASK_DATA_URL = "/tasks/{0}/attachments/{1}/contents"
+
+    @classmethod
+    def extract_text_from_bytes_data(cls, filename, file_data_bytes):
+        if filename.find('.pdf') != -1:
```

Comment:
* A lot of usage of `String.find()` to search for a sub-string
* We could make use of the `in` keyword to increase performance and readability
* See: https://stackoverflow.com/questions/18437798/find-vs-in-operation-in-string-python